src/Workspaces/CSharp/Portable/Simplification/Simplifiers/NameSimplifier.cs

```diff
@@ -0,0 +1,729 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeStyle;
+using Microsoft.CodeAnalysis.CSharp.Extensions;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Utilities;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Simplification;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Simplification.Simplifiers
+{
+    internal class NameSimplifier : AbstractCSharpSimplifier<NameSyntax, TypeSyntax>
```

Comment:
broke out code for simplifying names separate from simplifying expressions.  no need for such simliarly named methods to intermingle confusingly.